build.gradle.kts

```
@@ -170,24 +171,21 @@ buildConfig {
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // Dependencies
 
+listOf(libs.junit.jupiter, libs.assertj, libs.logback.classic).forEach {
+    dependencies {
+        testImplementation(it)
+        integrationTestImplementation(it)
+    }
+}
+
 dependencies {
```

Can we move this inside the `dependencies` block as follows?

```suggestion
dependencies {
    listOf(libs.junit.jupiter, libs.assertj, libs.logback.classic).forEach {
        testImplementation(it)
        integrationTestImplementation(it)
    }
```

---

updated with inclusion of the missing junit-platform-launcher